tests/load-tests/ci-scripts/utility_scripts/measure-time-skew.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Initialize sums
+sum_rtt=0
+sum_skew=0
+
+for i in {1..3}; do
+    # Step 1: Measure RTT in nanoseconds and calculate in seconds with fractional part
+    start=$(date +%s%N) # Start time in nanoseconds
+    oc exec prometheus-k8s-0 -n openshift-monitoring -- date -u +"%Y-%m-%dT%H:%M:%S%z" > /dev/null
+    end=$(date +%s%N)   # End time in nanoseconds
+
+    # Calculate round-trip time in seconds with fractions
+    rtt=$(echo "scale=6; ($end - $start) / 1000000000" | bc)
+    rtt=$(printf "%.3f" $rtt) # Round to three decimal places
```

Comment:
I think you can do this rounding on line above with `scale=3`

Reply:
Actually it might be better to not do any rounding here and rely o the rounding done down below when printing the values finally.

Reply:
I think we can update th upper line with scal=3 and remove the lower line since The final value of rtt will already be a floating-point number rounded to three decimal places